### DISCUSSION TITLE
Are Radeon cards supported on Linux?

### DISCUSSION THREAD

**Discussion by 64kramsystem** [bevyengine/bevy#2813] [Q&A] [answered]
I wrote a small pong game based on the pong example, which I test on a Linux machine.

Running it on an Nvidia card (proprietary drivers), there's no problem.

Running it on a Radeon card (RX 6600 XT, amdgpu-pro drivers), there are graphical issues.

Is it expected that a Bevy-based game runs the same way? When starting on Radeon, I see:

    WARNING: radv is not a conformant vulkan implementation, testing use only.

but it's not obvious (to me) if this is relevant or not.

**Comment by afonsolage** [comment, marked as answer]
This seems to be a problem with drivers amd drivers no implementing all vulkan features on linux, based on [my search](https://bbs.archlinux.org/viewtopic.php?id=268734)

**Reply by 64kramsystem** [reply]
Thanks. Sadly, attempting to run the OpenGL backend causes a segfault :exploding_head:.

**Reply by cart** [reply]
The OpenGL backend should work better once we switch to the new renderer in Bevy 0.6 and update to the latest wgpu.

**Comment by Ixentus** [comment]
Switch to regular amdgpu (non-pro) drivers. They work great

**Reply by 64kramsystem** [reply]
> Switch to regular amdgpu (non-pro) drivers. They work great

I think other game engines require the `amdgpu-pro`. At least, I remember starting with non-pro drivers and getting issues with some engines.

Also, I think that amdgpu-pro is something added on top of amdgpu, so in line of theory, it should still be possible to make amdgpu-pro work as they were amdgpu?

**Reply by bjorn3** [reply]
Based on https://wiki.archlinux.org/title/AMDGPU_PRO it seems that amdgpu-pro uses it's own libgl, replacing mesa, not just a different kernel driver. I wouldn't be surprised if it has worse Vulkan support.

**Reply by 64kramsystem** [reply]
Phew :sweat_smile: Thanks for the feedback! I'll replace with the standard driver, than have a better look at which engine has problems and how :cursing_face: :laughing:

**Reply by 64kramsystem** [reply]
The game still has corrupted graphics when using the amdgpu non-pro drivers (I didn't specify any Bevy flags) :cry:
